js_modules/dagster-ui/packages/ui-core/src/assets/LaunchAssetChoosePartitionsDialog.tsx

```diff
@@ -840,20 +849,17 @@ const PartitionSelectionNotice = ({
   onShowPreview?: () => void;
 }) => {
   return (
-    <Box padding={{horizontal: 16, bottom: 16}} border="bottom">
-      <Alert
-        intent="info"
-        title={
-          <Box flex={{gap: 12, alignItems: 'flex-start'}}>
-            <span>{text}</span>
-            {onShowPreview && (
-              <Button data-testid={testId('backfill-preview-button')} onClick={onShowPreview}>
-                Preview
-              </Button>
-            )}
-          </Box>
-        }
-      />
+    <Box padding={{horizontal: 16, top: 16, bottom: 8}} style={{position: 'relative'}} border="top">
+      <Alert intent="info" title={<Box style={{marginRight: 100, minHeight: 24}}>{text}</Box>} />
+      <div style={{position: 'absolute', top: 24, right: 24, zIndex: 4}}>
```

Comment:
This is a hack, but it's not possible to show a right-justified button inside an Alert using just the title + description props. if the text is short, the button floats in the middle of the alert box. 

I think in a follow-up we should simplify Alert and make it not use `Group` so that this isn't needed. Right now there are like 10 layers of divs in the alert so it's not really possible to wrap and adjust with styled-components either.

Reply:
oof but I get it